README.md

```diff
@@ -1,14 +1,13 @@
-# WARNING: This is a public repository!!!
+[NOTE!] This is a public repository.
```

Comment:
You may have to add a larger than character before [!NOTE] and move the text to the following line, also after a larger than character. I'm just learning about it too. https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts
```
> [NOTE!]
> Some message
```

Reply:
My linter really wanted to fight me on that one but fixed in 07b06fe.